cranelift-codegen/meta/src/shared/legalize.rs

```diff
@@ -449,21 +448,12 @@ pub(crate) fn define(insts: &InstructionGroup, imm: &Immediates) -> TransformGro
         }
     }
 
-    // Expansions for fcvt_from_{u,s}int for smaller integer types.
-    // These use expand and not widen because the controlling type variable for
-    // these instructions are f32/f64, which are legalized as part of the expand
+    // Expansion for fcvt_from_sint for smaller integer types.
+    // This use expand and not widen because the controlling type variable for
+    // this instruction is f32/f64, which is legalized as part of the expand
     // group.
     for &dest_ty in &[F32, F64] {
         for &src_ty in &[I8, I16] {
-            let bound_inst = fcvt_from_uint.bind(dest_ty).bind(src_ty);
-            expand.legalize(
-                def!(a = bound_inst(b)),
-                vec![
-                    def!(x = uextend.I32(b)),
-                    def!(a = fcvt_from_uint.dest_ty(x)),
-                ],
-            );
```

Comment:
To make sure I understand, this code isn't actually wrong; it's just not being used right now, right?

Reply:
That's right. (at least for x86, maybe other archs do use it) I can put it back if you want.

Reply:
Ok, cool. I don't think it's necessary to keep it right now. I expect we'll eventually want to refactor the x86 code to make even more of what `expand_fcvt_from_uint` does available to other architectures, but that's a bigger change and we can figure that out later.